### PR TITLE
Upper case token/chain codes from fio requests.

### PR DESCRIPTION
**Pull request by andreyvEze**

### CODE REVIEW

File: src/components/scenes/FioSentRequestDetailsScene.js [hunk 1]
```diff
@@ -53,12 +53,12 @@ export class FioSentRequestDetailsComponent extends React.Component<Props, Local
   }
 
   amountField = () => {
+    const { token_code: tokenCode, amount } = this.props.selectedFioSentRequest.content
```
Comment (andreyvEze) [inline src/components/scenes/FioSentRequestDetailsScene.js:56]: const fiatAmount = this.fiatAmount(tokenCode.toUpperCase(), amount)}

File: src/modules/FioAddress/util.js [hunk 2]
```diff
@@ -130,7 +130,7 @@ const isWalletConnected = async (
     const receiveAddress = await wallet.getReceiveAddress()
```
Comment (andreyvEze) [inline src/modules/FioAddress/util.js:130]: set tokenCode/chainCode to upper case in the beginning of the method

File: src/modules/FioAddress/util.js [hunk 3]
```diff
@@ -204,8 +204,8 @@ export const updatePubAddressesForFioAddress = async (
       isConnection: publicAddress && publicAddress !== '0'
     })
     iteration.publicAddressesToConnect.push({
```
Comment (andreyvEze) [inline src/modules/FioAddress/util.js:206]: set tokenCode/chainCode to upper case in the beginning of the method

File: src/modules/FioRequest/components/FioRequestRow.js [hunk 4]
```diff
@@ -177,15 +177,15 @@ const mapStateToProps = (state: RootState, ownProps: OwnProps) => {
...
   } catch (e) {
-    console.log('No denomination for this Token Code -', fioRequest.content.token_code)
+    console.log('No denomination for this Token Code -', fioRequest.content.token_code.toUpperCase())
   }
   const fiatSymbol = getFiatSymbol(wallet.fiatCurrencyCode)
   const isoFiatCurrencyCode = wallet.isoFiatCurrencyCode
   const exchangeRates = state.exchangeRates
 
-  const rateKey = `${fioRequest.content.token_code}_${isoFiatCurrencyCode}`
+  const rateKey = `${fioRequest.content.token_code.toUpperCase()}_${isoFiatCurrencyCode}`
```
Comment (andreyvEze) [inline src/modules/FioRequest/components/FioRequestRow.js:188]: once a the beginning of the method

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**